src/models/preparation.js

```diff
@@ -16,7 +22,7 @@ module.exports = (sequelize, DataTypes) => {
         primaryKey: true,
```

Comment:
Gotta get rid of the `primaryKey` that were on recipeId and diluentId

Reply:
Done.